tsconfig.json

```diff
@@ -8,7 +8,7 @@
     "module": "commonjs",                           /* Specify module code generation: 'none', 'commonjs', 'amd', 'system', 'umd', 'es2015', 'es2020', or 'ESNext'. */
     "lib": ["es6", "dom", "es2017"],                /* Specify library files to be included in the compilation. */
     "allowJs": true,                                /* Allow javascript files to be compiled. */
-    "types": ["cypress", "node", "jquery", "@testing-library/cypress"],
+    "types": ["node", "jquery", "./lib/jquery.jsonp-2.4.min", "cypress", "@testing-library/cypress"],
```

Comment:
I rearranged this a bit as it was picking up the `jquery` types in the `cypress` folder instead of the installed ones.  also not sure if there is a better way to deal with the `jquery.jsonp` as it's going to get a bit out of hand if we need to add all the `lib` types here separately as we go.